src/simian/mac/admin/__init__.py

```diff
@@ -221,8 +222,7 @@ def Render(self, template_path, values, write_to_response=True):
     path = os.path.join(
         os.path.dirname(__file__), 'templates', template_path)
 
-    if not settings.DEV_APPSERVER:
-      values['static_path'] = 'myapp/%s' % os.getenv('CURRENT_VERSION_ID')
+    values['static_path'] = 'myapp/%s' % os.getenv('CURRENT_VERSION_ID')
```

Comment:
With this conditional in place, the dev_appserver doesn't seem to serve up CSS etc. If I remove the conditional everything is fine.
I'm not the only one: https://groups.google.com/d/msg/simian-discuss/hRE2ieA_a14/9YoPqy6qeXAJ